hack/testing/logging.sh

```diff
@@ -119,10 +124,8 @@ trap "exit" INT TERM
 trap "cleanup" EXIT
 
 # override LOG_DIR and ARTIFACTS_DIR
-export LOG_DIR=${LOG_DIR:-${TMPDIR:-/tmp}/origin-aggregated-logging/logs}
-export ARTIFACT_DIR=${ARTIFACT_DIR:-${TMPDIR:-/tmp}/origin-aggregated-logging/artifacts}
 os::util::environment::use_sudo
-os::util::environment::setup_all_server_vars "origin-aggregated-logging/"
```

Comment:
we still need to call it, but without the `"origin-aggregated-logging/"` because that is set by `OS_TMP_ENV_SET` above.